universal-login-wallet/src/ui/react/Home/WelcomeScreen.tsx

```diff
@@ -1,18 +1,30 @@
-import React from 'react';
-import {useRouter} from '../../hooks';
+import React, {useState, useEffect} from 'react';
+import {useRouter, useServices} from '../../hooks';
+import {Notice} from '@universal-login/react';
+import {setBetaNotice} from '@universal-login/sdk';
 
 export const WelcomeScreen = () => {
   const {history} = useRouter();
+  const {sdk} = useServices();
+
+  const [notice, setNotice] = useState('');
+  useEffect(() => {
+    setBetaNotice(sdk);
```

Comment:
Move `setBetaNotice(sdk)` to `AppBootstraper` initialization code (right near `sdk.start()`).

And then this just becomes: `const notice = sdk.getNotice()`

Reply:
👍